demos/object_detection_demo_ssd_async/main.cpp

```diff
@@ -50,523 +135,127 @@ bool ParseAndCheckCommandLine(int argc, char *argv[]) {
     return true;
 }
 
-void frameToBlob(const cv::Mat& frame,
-                 InferRequest::Ptr& inferRequest,
-                 const std::string& inputName) {
-    if (FLAGS_auto_resize) {
-        /* Just set input blob containing read image. Resize and layout conversion will be done automatically */
-        inferRequest->SetBlob(inputName, wrapMat2Blob(frame));
-    } else {
-        /* Resize and copy data from the image to the input blob */
-        Blob::Ptr frameBlob = inferRequest->GetBlob(inputName);
-        matU8ToBlob<uint8_t>(frame, frameBlob);
-    }
-}
+void paintInfo(cv::Mat& frame, const PipelineBase::PerformanceInfo& info) {
+    std::ostringstream out;
 
-enum class ExecutionMode {USER_SPECIFIED, MIN_LATENCY};
+    out.str("");
+    out << "FPS:" << std::fixed << std::setprecision(0) << std::setw(3) << info.movingAverageFPS
+        << " (" << std::setprecision(1) << info.FPS << ")";
```

Comment:
Reporting two values is confusing.

Reply:
1st one is moving average (immediate value), second one is average. It's pretty clear if you look at values. I don't wat to add additional text there as labels are already quite large, but we need both values here

Reply:
I don't understand how I can infer which value corresponds to which metric just by looking at them.

The suggestion is not to report average.